### PR TITLE
RUN-5253 change export type to es modules

### PR DESCRIPTION
**Pull request by pbaize**
#### Description of Change
Changes exports in the core from using module.export to named exports. Required for typescript 3 upgrade

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title starts with the JIRA ticket [pull request process](https://github.com/openfin/Internal-Wiki/wiki/Pull-Request-Process)

#### Release Notes

Notes: n/a

### CODE REVIEW

File: src/browser/deferred.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { BrowserWindow as BrowserWindowOF } from '../shapes';
 import { toSafeInt } from '../common/safe_int';
 import { clipBounds } from './utils';
 
-export { handleMove };
+export default { handleMove };
```
Comment (whyn07m3) [inline src/browser/deferred.ts]: we can't remove this, can we?

Comment (pbaize) [reply src/browser/deferred.ts]: how so?

Comment (whyn07m3) [reply src/browser/deferred.ts]: Now, we do `import { handleMove } from './deferred';`. But we already `export function handleMove(` export this function. So if we remove this default export of a map, the imports won't change. It looks like an extra line to me, hope I'm not missing something

Comment (pbaize) [reply src/browser/deferred.ts]: It might be extra, left it in for paranoia. IMO it's never a bad thing to provide a default export of all named exports

Comment (whyn07m3) [reply src/browser/deferred.ts]: I'm 100% fine if you know it's good to have it there. I'm not okay if paranoia wrote this line of code :)

File: src/browser/api_protocol/api_handlers/application.js [hunk 2]
```diff
@@ -66,7 +66,7 @@ module.exports.applicationApiMap = {
     'wait-for-hung-application': waitForHungApplication
 };
 
-module.exports.init = function() {
+export const init = function() {
     apiProtocolBase.registerActionMap(module.exports.applicationApiMap, 'Application');
```
Comment (MichaelMCoates) [inline src/browser/api_protocol/api_handlers/application.js]: `module.exports` left over.

**Pull request merged**